server/src/shared/genanki/genanki.test.ts

```diff
@@ -3,8 +3,7 @@ import { Package } from "./package";
 import { Deck } from "./deck";
 import { Note } from "./note";
 import { defaultModel } from "./default";
-import fetch from "node-fetch";
-
+const fetch = require("node-fetch");
```

Comment:
I was getting an error related to the `node-fetch` package when I used "import", but the package seems to work when using "require"